### PR TITLE
add findPath

### PR DESCRIPTION
**Pull request by hongchuan97**
add find path
Print out all the transfer bus stop code
Still need to clean up

### CODE REVIEW

File: src/main/java/duke/commands/Command.java [hunk 1]
```diff
@@ -14,5 +16,5 @@
      * @param ui The user interface displaying events on the task list.
      * @param storage The duke.storage object containing task list.
      */
-    public abstract void execute(Ui ui, Storage storage) throws DukeException;
+    public abstract void execute(Ui ui, Storage storage) throws DukeException, IOException;
```
Comment (Jefferson111) [inline src/main/java/duke/commands/Command.java]: IOException is no longer thrown

File: src/main/java/duke/Duke.java [hunk 2]
```diff
@@ -41,7 +43,7 @@ public void getResponse(String userInput) {
             if (c instanceof ExitCommand) {
                 tryExitApp();
             }
-        } catch (DukeException e) {
+        } catch (DukeException | IOException e) {
```
Comment (Jefferson111) [inline src/main/java/duke/Duke.java]: Exception is no longer thrown

File: src/main/java/duke/commands/GetBusRouteCommand.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
+package duke.commands;
+
+import duke.commons.DukeException;
```
Comment (Jefferson111) [inline src/main/java/duke/commands/GetBusRouteCommand.java]: Currently is duke.commons.exceptions.DukeExceptions

File: src/main/java/duke/Duke.java (hunk 2 above)
Comment (hongchuan97) [reply src/main/java/duke/Duke.java]: Done

File: src/main/java/duke/commands/GetBusRouteCommand.java (hunk 3 above)
Comment (hongchuan97) [reply src/main/java/duke/commands/GetBusRouteCommand.java]: Done

File: src/main/java/duke/commands/Command.java (hunk 1 above)
Comment (hongchuan97) [reply src/main/java/duke/commands/Command.java]: Done

**Pull request merged**